doc/sphinx/proof-engine/tactics.rst

```diff
@@ -71,22 +74,22 @@ specified, the default selector is used.
 Bindings list
 ~~~~~~~~~~~~~~~~~~~
 
-Tactics that take a term as argument may also support a bindings list,
+Tactics that take a term as an argument may also support a bindings list,
 so as to instantiate some parameters of the term by name or position.
 The general form of a term equipped with a bindings list is ``term with
 bindings_list`` where ``bindings_list`` may be of two different forms:
 
 .. _bindings_list_grammar:
 
   .. productionlist:: `bindings_list`
-     bindings_list : (ref := `term`) ... (ref := `term`)
+     bindings_list : (`ref` := `term`) ... (`ref` := `term`)
                    : `term` ... `term`
 
-+ In a bindings list of the form :n:`{* (ref:= term)}`, :n:`ref` is either an
++ In a bindings list of the form :n:`{+ (@ref:= @term)}`, :n:`@ref` is either an
```

Comment:
If `ref` was never defined in a grammar, it would be worth adding it to the grammar above.

Reply:
I added:

```
     ref           : ident
                   : INT
```

And I noticed that "INT" is a poor name, since it only includes natural numbers.  Maybe I'll clean that up in another PR.